content/integrations/hipchat.md

```diff
@@ -28,7 +28,10 @@ Tick the checkbox if you want to be notified for every comment, in all configure
 
 You can now share graphs or send alerts to HipChat rooms using the syntax @hipchat-chat_name
 
-**NOTICE** : If your chat name contains special characters such as commas or brackets, they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.
+**NOTICE** : 
+If you are using a hipchat API V1 token, if your chat name contains special characters such as commas or brackets, they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.
```

Comment:
It's when you want to send a notification from a monitor, you would do @my_hipchat_handle , but some HipChat handles can contain some special characters that we escape automatically in-app, to not confuse our users we display this notice here. 

But apparently this auto-escaping is only working with API V1, hence the update